### DISCUSSION TITLE
callbackUrl=https%3A%2F%2Fmoes.pdev.focusbrands.com%2Faccount%2Fsignin&error=OAuthCallback#_=_

### DISCUSSION THREAD

**Discussion by hemanthkumar-govindan** [nextauthjs/next-auth#9395] [Help]
I have a app setup with next-auth for Facebook and Apple providers and is deployed to vercel domain. When i try to login to Apple or Facebook I am able to go through the complete login flow but after that redirecting back to app is failing with error.

callbackUrl=https%3A%2F%2Fmoes.pdev.focusbrands.com%2Faccount%2Fsignin&error=OAuthCallback#_=_

Anyone can help please.

**Comment by leo3linbeck** [comment]
Discussed here: https://github.com/nextauthjs/next-auth/discussions/9942. Cheers, L3
